src/core/dbus-cgroup.c

```diff
@@ -2053,6 +2054,38 @@ int bus_cgroup_set_property(
                 return 1;
         }
 
+        if (streq(name, "ManagedOOMMemoryPressureDurationUSec")) {
+                uint64_t t;
+
+                if (!UNIT_VTABLE(u)->can_set_managed_oom)
+                        return sd_bus_error_setf(error, SD_BUS_ERROR_INVALID_ARGS, "Cannot set %s for this unit type", name);
+
+                r = sd_bus_message_read(message, "t", &t);
+                if (r < 0)
+                        return r;
+
+                /* Zero is treated as default value (USEC_INFINITY) in D-Bus API. */
+                t = t ?: USEC_INFINITY;
+
+                if (t < 1 * USEC_PER_SEC)
+                        return sd_bus_error_setf(error, SD_BUS_ERROR_INVALID_ARGS, "%s= must be at least 1s, got %" PRIu64, name, t);
```

Comment:
Thanks! I used FORMAT_TIMESPAN with accuracy = USEC_PER_SEC since accuracy = 1 (aka second resolution) doesn't seem very useful when we're printing a value less than 1 second. Let me know if you prefer millisecond resolution

Reply:
I do not have any strong opinion about this.